core/tests/test_utils.py

```diff
@@ -1251,17 +1276,40 @@ def assertRaises(self, *args, **kwargs):
             'self.assertRaises should not be used in these tests. Please use '
             'self.assertRaisesRegexp instead.')
 
-    def assertRaisesRegexp(  # pylint: disable=keyword-arg-before-vararg
-            self, expected_exception, expected_regexp, callable_obj=None,
-            *args, **kwargs):
-        if not expected_regexp:
+    def assertRaisesRegexp(  # pylint: disable=invalid-name
+            self, expected_exception, expected_regex, *args, **kwargs):
+        """Asserts that the message in a raised exception matches a regex.
+        This is a wrapper around assertRaisesRegex in unittest that enforces
+        strong regex.
+
+        Args:
+            expected_exception: Exception. Exception class expected
+                to be raised.
+            expected_regex: re.Pattern|str. Regex expected to be found in
+                error message.
+            *args: list(*). Function to be called and extra positional args.
+            **kwargs: dict(str, Any). Extra kwargs.
+
+        Returns:
+            bool. Whether the code raised exception in the expected format.
+        """
+        if not expected_regex:
             raise Exception(
                 'Please provide a sufficiently strong regexp string to '
                 'validate that the correct error is being raised.')
 
-        return super(TestBase, self).assertRaisesRegexp(
-            expected_exception, expected_regexp,
-            callable_obj=callable_obj, *args, **kwargs)
+        return super(TestBase, self).assertRaisesRegex(
+            expected_exception, expected_regex, *args, **kwargs)
+
+    def assertItemsEqual(self, *args, **kwargs):  # pylint: disable=invalid-name
+        """An unordered sequence comparison asserting that the same elements,
```

Comment:
First "sentence" is a sentence fragment.

Reply:
Rephrased.